tests/trnsysGUI/storageTank/testConfigureStorageDialog.py

```diff
@@ -15,26 +17,28 @@
 class TestConfigureStorageDialog:
     storageTank = None
 
-    def triggerStorageDialog(self, qtbot):
+    @pytest.fixture()
+    def triggerStorageDialog(
+        self, qtbot, monkeypatch
+    ) -> ConfigureStorageDialog:
+        monkeypatch.setattr(QMessageBox, "exec", lambda self: QMessageBox.Ok)
+
         mainWindow = utils.createMainWindow(_PROJECT_DIR, _PROJECT_NAME)
         qtbot.addWidget(mainWindow)
         self.storageTank = utils.getDesiredTrnsysObjectFromList(
-            mainWindow.editor.trnsysObj, stw.StorageTank
-        )
-        configureStorageDialog = self.storageTank.mouseDoubleClickEvent(
-            _core.QEvent.MouseButtonDblClick, isTest=True
```

Comment:
Nice choice. Dealing with the dialog directly keeps this GUI part out of these tests. When we start separating the GUI code properly, these tests should stay valid and the "doubleclickeEvent" would be a minimal GUI test (not for now).